### DISCUSSION TITLE
New version release on plan?

### DISCUSSION THREAD

**Discussion by neuromaancer** [LazyVim/LazyVim#2577] [Q&A] [answered]
It's been a while there isn't a new release but more and more pull requests. Is there any plan for a new release?

**Comment by 0x2E** [comment]
- https://github.com/LazyVim/LazyVim/issues/1977
- https://twitter.com/Folke/status/1748244437908504860

**Reply by neuromaancer** [reply]
Yeah, but this is like a month ago😅

**Comment by skoch13** [comment, marked as answer]
Unfortunately, the same happened to all of the folke's repos/projects lately. I understand the "OSS maintainer owes you nothing" idea, but having additional maintainers could be a good option too. This was already [asked](https://github.com/LazyVim/LazyVim/issues/1977#issuecomment-1846857911) about, but there was no subsequent reaction from the main maintainer's side.

Still, I cannot imagine how much effort and willpower it takes to create and keep up-to-date such a large number of plugins.

**Reply by neuromaancer** [reply]
I think he might wait for the nvim 0.10 to be released. 😏

**Reply by skoch13** [reply]
yeah, I second that, but on the flip side one can just be tired 🤷

**Comment by dusty-phillips** [comment]
I have been toying with the idea of maintaining a fork, but I’m not sure I have the time to do it justice. folke’s intuition for excellence is not an easy thing to emulate.
